services/types.go

```diff
@@ -3,4 +3,5 @@ package services
 type OnboardingTicket struct {
 	ID             string `json:"id"`
 	ExpirationDate int64  `json:"expirationDate,string"`
+	Quota          string `json:"quota"`
```

Comment:
where does this quota get validated with the existing quota in ceph? or we don't have any plan to consider the ceph quota and users can set whatever quota they want?

Reply:
which means we can use quota more than the ceph cluster on all the consumer clusters.

Reply:
The quota on the onboarding token cannot be validated as generating an onboarding token is a customer-owned operation. What we can do we can decide to not allow onboarding using this token if the quota does not make sense

Reply:
that make sense, this is only to generate the token and we are going to have followup PR to block such tokens with quota, please mark this as resolved as i don't have permission to do it.

Reply:
@rchikatw  please take note to issue a follow-up PR with the needed changes.